.nvmrc

```diff
@@ -0,0 +1 @@
+14.21.3
```

Comment:
I would recommend we all use NVM (Node Version Manager) with a shell addition to automatically switch to the project version of Node when switching into any directory in the project. Either way this file makes that possible and is how I run in my dev environment: https://github.com/nvm-sh/nvm